<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="game_name">
     <property name="font">
      <font>
       <family>Console</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Alien vs. Manse: The Reckoning</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="choose_difficulty_text">
     <property name="font">
      <font>
       <family>Console</family>
      </font>
     </property>
     <property name="text">
      <string>Choose game difficulty:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTextBrowser" name="difficulty_text"/>
   </item>
   <item row="3" column="0">
    <widget class="QSpinBox" name="choose_difficulty_spinbox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
